android/exponentview/src/main/java/versioned/host/exp/exponent/modules/api/notifications/NotificationsHelper.java

```diff
@@ -0,0 +1,198 @@
+package versioned.host.exp.exponent.modules.api.notifications;
```

Comment:
can you move this out of the `versioned` directory? just put it under exponentview/host.exp.exponent